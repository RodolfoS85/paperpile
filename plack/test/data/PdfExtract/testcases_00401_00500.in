---
file: ../../../PDFREPOSITORY/000401.pdf
---
file: ../../../PDFREPOSITORY/000402.pdf
---
file: ../../../PDFREPOSITORY/000403.pdf
---
file: ../../../PDFREPOSITORY/000404.pdf
---
file: ../../../PDFREPOSITORY/000405.pdf
---
file: ../../../PDFREPOSITORY/000406.pdf
---
file: ../../../PDFREPOSITORY/000407.pdf
---
file: ../../../PDFREPOSITORY/000408.pdf
---
file: ../../../PDFREPOSITORY/000409.pdf
---
file: ../../../PDFREPOSITORY/000410.pdf
---
file: ../../../PDFREPOSITORY/000411.pdf
---
file: ../../../PDFREPOSITORY/000412.pdf
---
file: ../../../PDFREPOSITORY/000413.pdf
---
file: ../../../PDFREPOSITORY/000414.pdf
---
file: ../../../PDFREPOSITORY/000415.pdf
---
file: ../../../PDFREPOSITORY/000416.pdf
---
file: ../../../PDFREPOSITORY/000417.pdf
---
file: ../../../PDFREPOSITORY/000418.pdf
---
file: ../../../PDFREPOSITORY/000419.pdf
---
file: ../../../PDFREPOSITORY/000420.pdf
---
file: ../../../PDFREPOSITORY/000421.pdf
---
file: ../../../PDFREPOSITORY/000422.pdf
---
file: ../../../PDFREPOSITORY/000423.pdf
---
file: ../../../PDFREPOSITORY/000424.pdf
---
file: ../../../PDFREPOSITORY/000425.pdf
---
file: ../../../PDFREPOSITORY/000426.pdf
---
file: ../../../PDFREPOSITORY/000427.pdf
---
file: ../../../PDFREPOSITORY/000428.pdf
---
file: ../../../PDFREPOSITORY/000429.pdf
---
file: ../../../PDFREPOSITORY/000430.pdf
---
file: ../../../PDFREPOSITORY/000431.pdf
---
file: ../../../PDFREPOSITORY/000432.pdf
---
file: ../../../PDFREPOSITORY/000433.pdf
---
file: ../../../PDFREPOSITORY/000434.pdf
---
file: ../../../PDFREPOSITORY/000435.pdf
---
file: ../../../PDFREPOSITORY/000436.pdf
---
file: ../../../PDFREPOSITORY/000437.pdf
---
file: ../../../PDFREPOSITORY/000438.pdf
---
file: ../../../PDFREPOSITORY/000439.pdf
---
file: ../../../PDFREPOSITORY/000440.pdf
---
file: ../../../PDFREPOSITORY/000441.pdf
---
file: ../../../PDFREPOSITORY/000442.pdf
---
file: ../../../PDFREPOSITORY/000443.pdf
---
file: ../../../PDFREPOSITORY/000444.pdf
---
file: ../../../PDFREPOSITORY/000445.pdf
---
file: ../../../PDFREPOSITORY/000446.pdf
---
file: ../../../PDFREPOSITORY/000447.pdf
---
file: ../../../PDFREPOSITORY/000448.pdf
---
file: ../../../PDFREPOSITORY/000449.pdf
---
file: ../../../PDFREPOSITORY/000450.pdf
---
file: ../../../PDFREPOSITORY/000451.pdf
---
file: ../../../PDFREPOSITORY/000452.pdf
---
file: ../../../PDFREPOSITORY/000453.pdf
---
file: ../../../PDFREPOSITORY/000454.pdf
---
file: ../../../PDFREPOSITORY/000455.pdf
---
file: ../../../PDFREPOSITORY/000456.pdf
---
file: ../../../PDFREPOSITORY/000457.pdf
---
file: ../../../PDFREPOSITORY/000458.pdf
---
file: ../../../PDFREPOSITORY/000459.pdf
---
file: ../../../PDFREPOSITORY/000460.pdf
---
file: ../../../PDFREPOSITORY/000461.pdf
---
file: ../../../PDFREPOSITORY/000462.pdf
---
file: ../../../PDFREPOSITORY/000463.pdf
---
file: ../../../PDFREPOSITORY/000464.pdf
---
file: ../../../PDFREPOSITORY/000465.pdf
---
file: ../../../PDFREPOSITORY/000466.pdf
---
file: ../../../PDFREPOSITORY/000467.pdf
---
file: ../../../PDFREPOSITORY/000468.pdf
---
file: ../../../PDFREPOSITORY/000469.pdf
---
file: ../../../PDFREPOSITORY/000470.pdf
---
file: ../../../PDFREPOSITORY/000471.pdf
---
file: ../../../PDFREPOSITORY/000472.pdf
---
file: ../../../PDFREPOSITORY/000473.pdf
---
file: ../../../PDFREPOSITORY/000474.pdf
---
file: ../../../PDFREPOSITORY/000475.pdf
---
file: ../../../PDFREPOSITORY/000476.pdf
---
file: ../../../PDFREPOSITORY/000477.pdf
---
file: ../../../PDFREPOSITORY/000478.pdf
---
file: ../../../PDFREPOSITORY/000479.pdf
---
file: ../../../PDFREPOSITORY/000480.pdf
---
file: ../../../PDFREPOSITORY/000481.pdf
---
file: ../../../PDFREPOSITORY/000482.pdf
---
file: ../../../PDFREPOSITORY/000483.pdf
---
file: ../../../PDFREPOSITORY/000484.pdf
---
file: ../../../PDFREPOSITORY/000485.pdf
---
file: ../../../PDFREPOSITORY/000486.pdf
---
file: ../../../PDFREPOSITORY/000487.pdf
---
file: ../../../PDFREPOSITORY/000488.pdf
---
file: ../../../PDFREPOSITORY/000489.pdf
---
file: ../../../PDFREPOSITORY/000490.pdf
---
file: ../../../PDFREPOSITORY/000491.pdf
---
file: ../../../PDFREPOSITORY/000492.pdf
---
file: ../../../PDFREPOSITORY/000493.pdf
---
file: ../../../PDFREPOSITORY/000494.pdf
---
file: ../../../PDFREPOSITORY/000495.pdf
---
file: ../../../PDFREPOSITORY/000496.pdf
---
file: ../../../PDFREPOSITORY/000497.pdf
---
file: ../../../PDFREPOSITORY/000498.pdf
---
file: ../../../PDFREPOSITORY/000499.pdf
---
file: ../../../PDFREPOSITORY/000500.pdf
