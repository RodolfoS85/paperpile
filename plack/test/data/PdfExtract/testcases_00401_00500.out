---
authors: 'Foissac, Sylvain and Schiex, Thomas'
doi: 10.1186/1471-2105-6-25
title: Integrating alternative splicing detection into gene prediction
---
authors: 'Fontana, Walter and Schuster, Peter'
title: 'Shaping Space: The Possible and the Attainable in RNA Genotype-Phenotype Mapping'
---
authors: 'Force, Allan and Lynch, Michael and Pickett, F. Bryan and Amores, Angel and Yan, Yi-lin and Postlethwait, John'
title: 'Preservation of Duplicate Genes by Complementary, Degenerative Mutations'
---
authors: 'Forrest, Lucy R. and Tang, Christopher L. and Honig, Barry'
doi: 10.1529/biophysj.106.082313
title: On the Accuracy of Homology Modeling and Sequence Alignment Methods Applied to Membrane Proteins
---
authors: 'Fox-Walsh, Kristi L. and Dou, Yimeng and Lam, Bianca J. and Hung, She-pin and Baldi, Pierre F. and Hertel, Klemens J.'
doi: 10.1073/pnas.0508489102
title: The architecture of pre-mRNAs affects mechanisms of splice-site pairing
---
authors: 'Frantz, S Imon A. and Thiara, Amrik S. and Lodwick, David and Ng, Leong L. and Eperon, Ian C. and Samani, Nilesh J.'
title: Exon repetition in mRNA
---
authors: 'Freyhult, Eva K. and Bollback, Jonathan P. and Gardner, Paul P.'
doi: 10.1101/gr.5890907
title: 'Exploring genomic dark matter: A critical assessment of the performance of homology search methods on noncoding RNA'
---
authors: 'Friedman, Nir'
title: Discretizing Continuous Attributes While Learning Bayesian Networks
---
authors: 'Friedman, Robin C. and Farh, Kyle Kai-How and Burge, Christopher B. and Bartel, David P.'
doi: 10.1101/gr.082701.108
title: Most mammalian mRNAs are conserved targets of microRNAs
---
authors: 'Friedman, Nir and Linial, Michal'
title: Using Bayesian Networks to Analyze Expression Data
---
authors: 'Friedrich, Markus'
title: 'Evolution of Insect Eye Development: First Insights from Fruit Fly, Grasshopper and Flour Beetle'
---
authors: 'Frith, Martin C. and Hansen, Ulla and Weng, Zhiping'
title: Detection of cis-element clusters in higher eukaryotic DNA
---
authors: 'Fuhrman, Jed'
title: Genome sequences from the sea
---
authors: 'Fujii, Yoshifumi and Shimizu, Toshiyuki and Toda, Takashi and Yanagida, Mitsuhiro and Hakoshima, Toshio'
title: Structural basis for the diversity of DNA recognition by bZIP transcription factors
---
authors: 'Fung, Eric T and Thulasiraman, Vanitha and Weinberger, Scot R and Dalmasso, Enrique A'
title: Protein biochips for differential profiling
---
authors: 'Furlanello, Cesare and Serafini, Maria and Merler, Stefano and Jurman, Giuseppe'
title: Entropy-based gene ranking without selection bias for the predictive classification of microarray data
---
authors: 'Furnham, Nicholas and Ruffle, Stuart and Southan, Christopher'
title: 'Splice Variants: A Homology Modeling Approach'
---
authors: 'Futcher, Bruce'
title: Microarrays and cell cycle transcription in yeast
---
authors: 'Gabrielian, Andrei E. and Landsman, David and Bolshoy, Alexander'
title: Curved DNA in promoter sequences
---
authors: 'Gadiraju, Sashidhar and Vyhlidal, Carrie A and Leeder, J Steven and Rogan, Peter K'
title: 'Genome-wide prediction, display and refinement of binding sites with information theory-based models'
---
authors: 'Gaidatzis, Dimos and van Nimwegen, Erik and Hausser, Jean and Zavolan, Mihaela'
doi: 10.1186/1471-2105-8-69
title: Inference of miRNA targets using evolutionary conservation and pathway analysis
---
authors: 'Gailus-Durner, Valerie and Scherf, Matthias and Werner, Thomas'
doi: 10.1007/s003350010219
title: Experimental data of a single promoter can be used for in silico detection of genes with related regulation in the absence of sequence similarity
---
authors: 'Galante, Pedro Alexandre Favoretto and Sakabe, Noboru JO and KIRSCHBAUM-SLAGER, Natanja and JOSÉ DE SOUZA, Sandro'
doi: 10.1261/rna.5123504
title: Detection and evaluation of intron retention events in the human transcriptome
---
authors: 'Gane, Paul J and Dean, Philip M'
title: Recent advances in structure-based rational drug design
---
authors: 'Gao, Feng and Foat, Barrett C and Bussemaker, Harmen J'
title: Defining transcriptional networks through integrative modeling of mRNA expression and transcription factor binding data
---
authors: 'Garcia-Blanco, Mariano A and Baraniak, Andrew P and Lasda, Erika L'
title: Alternative splicing in disease and therapy
---
authors: 'Gardner, Paul P and Giegerich, Robert'
doi: 10.1186/1471-2105-5-140
title: A comprehensive comparison of comparative RNA structure prediction approaches
---
authors: 'Gaspin, Christine'
title: RNA Secondary Structure Determination and Representation Based on Constraints Satisfaction
---
authors: 'Gautheret, Daniel and Lambert, Andre'
doi: 10.1006/jmbi.2001.5102
title: Direct RNA Motif Definition and Identification from Multiple Sequence Alignments using Secondary Structure Profiles
---
authors: 'Gautheret, Daniel and Poirot, Olivier and Lopez, Fabrice and Audic, Stephane and Claverie, Jean-Michel'
title: 'Alternate Polyadenylation in Human mRNAs: A Large-Scale Analysis by EST Clustering'
---
authors: 'Gazzoli, Isabella and Kolodner, Richard D.'
doi: 10.1128/MCB.23.22.7992-8007.2003
title: Regulation of the Human MSH6 Gene by the Sp1 Transcription Factor and Alteration of Promoter Activity and Expression by Polymorphisms
---
authors: 'Geisinger, Edward and Adhikari, Rajan P. and Jin, Ruzhong and Ross, Hope F. and Novick, Richard P.'
doi: 10.1111/j.1365-2958.2006.05292.x
title: 'Inhibition of rot translation by RNAIII, a key feature of agr function'
---
authors: 'Gelfand, M. S. and Koonin, E. V. and Mironov, A. A.'
title: Prediction of transcription regulatory sites in Archaea by a comparative genomic approach
---
authors: 'Gelfand, M Ikhail S. and Mironov, Andrey A. and Pevzner, Pavel A.'
title: Gene recognition via spliced sequence alignment
---
authors: 'Gent, Ian P. and Harvey, Warwick and Kelsey, Tom'
title: 'Groups and Constraints: Symmetry Breaking during Search'
---
authors: 'Ghimire, Ganga D. and Imai, Kenichiro and Akazawa, Fumitsugu and Tsuji, Toshiyuki and Sonoyama, Masashi and Mitaku, Shigeki'
title: Physicochemical properties of amino acid sequences of G-proteins for understanding GPCR-G-protein coupling
---
authors: 'Ghirlando, Rodolfo and Trainor, Cecelia D.'
doi: 10.1074/jbc.M002053200
title: GATA-1 Bends DNA in a Site-independent Fashion
---
authors: 'Ghosh, Sagarmoy and GARCIA-BLANCO, Mariano A.'
title: Coupled in vitro synthesis and splicing of RNA polymerase II transcripts
---
authors: 'Ghosh, David'
title: Object-oriented Transcription Factors Database (ooTFD)
---
authors: 'Ghoshal, Kalpana and Datta, Jharna and Majumder, Sarmila and Bai, Shoumei and Dong, Xiaocheng and Parthun, Mark and Jacob, Samson T.'
doi: 10.1128/MCB.22.23.8302-8319.2002
title: Inhibitors of Histone Deacetylase and DNA Methyltransferase Synergistically Activate the Methylated Metallothionein I Promoter by Activating the Transcription Factor MTF-1 and Forming an Open Chromatin Structure
---
authors: '{Rat Genome Sequencing Project Consortium}'
title: Genome sequence of the Brown Norway rat yields insights into mammalian evolution
---
authors: 'Gibson, Michael Andrew and Mjolsness, Eric'
title: Modeling the Activity of Single Genes
---
authors: 'Giedroc, David P. and Theimer, Carla A. and Nixon, Paul L.'
doi: 10.1006/jmbi.2000.3668
title: 'Structure, Stability and Function of RNA Pseudoknots Involved in Stimulating Ribosomal Frameshifting'
---
authors: 'Giegerich, Robert'
title: PREDICTION AND VISUALIZATION OF STRUCTURAL SWITCHES IN RNA
---
authors: 'Giegerich, Robert and Meyer, Carsten and Steffen, Peter'
title: Towards a Discipline of Dynamic Programming
---
authors: 'Giegerich, Robert'
title: A Systematic Approach to Dynamic Programming in Bioinformatics
---
authors: 'Gilbert, David'
title: Introduction to the Special Issue on Bioinformatics
---
authors: 'Gilbert, David and Westhead, David and Nagano, Nozomi and Thornton, Janet'
title: Motif-based searching in TOPS protein topology databases
---
authors: 'Gilbert, David and Westhead, David and Viksna, Juris and Thornton, Janet'
title: A computer system to perform structure comparison using TOPS representations of protein structure
---
authors: 'Ginsberg, Michael D. and Feliciello, Antonio and Jones, Jeffrey K. and Avvedimento, Enrico V. and Gottesman, Max E.'
doi: 10.1016/S0022-28360300173-6
title: PKA-dependent Binding of mRNA to the Mitochondrial AKAP121 Protein
---
authors: 'Gobel, Ulrike and Sander, Chris and Schneider, Reinhard and Valencia, Alfonso'
title: Correlated Mutations and Residue Contacts in Proteins
---
authors: 'Goericke, Ralf and Welschmeyer, a nd Nicholas A.'
title: The ma rin e prochlorophyte Proc h loroco c c us contributes significantly to p hyt op lankt on biomass and primary production in the Sargasso Sea
---
authors: 'Goguel, Valerie and {t YUE WANG} and ROSBASHl, AD Michael'
title: Short Artificial Hairpins Sequester Splicing Signals and Inhibit Yeast Pre-mRNA Splicing
---
authors: 'Gontarek, Richard R. and Derse, David'
title: 'Interactions among SR Proteins, an Exonic Splicing Enhancer, and a Lentivirus Rev Protein Regulate Alternative Splicing'
---
authors: 'González-Díaz, Humberto and González-Díaz, Yenny and Santana, Lourdes and Ubeira, Florencio M. and Uriarte, Eugenio'
doi: 10.1002/pmic.200700638
title: 'Proteomics, networks and connectivity indices'
---
authors: 'Gonzalez-Herrera, I. G. and Prado-Lourenco, L. and Teshima-Kondo, S. and Kondo, K. and Cabon, F. and -F. Arnal, J. and Bayard, F. and -C. Prats, A.'
title: IRES-dependent regulation of FGF-2 mRNA translation in pathophysiological conditions in the mouse
---
authors: 'Good, L.'
doi: 10.1007/s00018-003-3045-4
title: Translation repression by antisense sequences
---
authors: 'Gooding, Clare and Clark, Francis and Wollerton, Matthew C and Grellscheid, Sushma-Nagaraja and Groom, Harriet and Smith, Christopher WJ'
doi: 10.1186/gb-2006-7-1-r1
title: A class of human exons with predicted distant branch points revealed by analysis of AG dinucleotide exclusion zones
---
authors: 'Gordon, J. J. and Towsey, M. W. and Hogan, J. M. and Mathews, S. A. and Timms, P.'
title: Improved Prediction of Bacterial Transcription Start Sites
---
authors: 'Gorin, Andrey A. and Zhurkin, Victor B. and Olson, Wilma K.'
title: B-DNA Twisting Correlates with Base-pair Morphology
---
authors: 'Gorodkin, Jan and Lyngsø, Rune B. and Stormo, Gary D.'
title: A Mini-Greedy Algorithm for Faster Structural RNA Stem-Loop Search
---
authors: 'Gorodkin, J. and Stærfeldt, H. H. and Lund, O. and Brunak, S.'
title: 'MatrixPlot: visualizing sequence constraints'
---
authors: 'Gorodkin, J. and Zwieb, C. and Knudsen, B.'
title: Semi-automated update and cleanup of structural RNA alignment databases
---
authors: 'Gorodkin, J. and Heyer, L. J. and Stormo, G. D.'
title: Finding the most significant common sequence and structure motifs in a set of RNA sequences
---
authors: 'Gottesman, Susan'
doi: 10.1016/j.tig.2005.05.008
title: 'Micros for microbes: non-coding regulatory RNAs in bacteria'
---
authors: 'Göttgens, Berthold and Barton, Linda M. and Gilbert, James G. R. and Bench, Anthony J. and Sanchez, Maria-José and Bahn, Sabine and Mistry, Shailesh and Grafham, Darren and McMurray, Amanda and Vaudin, Mark and Amaya, Enrique and Bentley, David R. and Green, Anthony R.'
title: Analysis of vertebrate SCL loci identifies conserved enhancers
---
authors: 'Govindarajan, Sridhar and Goldstein, Richard A.'
title: Evolution of Model Proteins on a Foldability Landscape
---
authors: 'RIDHAR GOVINDARAJAN, S and Goldstein, Richard A.'
title: On the thermodynamic hypothesis of protein folding
---
authors: 'Govindarajan, Sridhar and Recabarren, Ruben and Goldstein, Richard A.'
title: Estimating the Total Number of Protein Folds
---
authors: 'Grabe, Niels'
title: 'AliBaba2: Context Specific Identification of Transcription Factor Binding Sites'
---
authors: 'Grabowski, Paula'
title: Alternative splicing in parallel
---
authors: 'Grasso, C. and Modrek, B. and Xing, Y. and Lee, C.'
title: GENOME-WIDE DETECTION OF ALTERNATIVE SPLICING IN EXPRESSED SEQUENCES USING PARTIAL ORDER MULTIPLE SEQUENCE ALIGNMENT GRAPHS
---
authors: 'Graveley, Brenton R. and Hertel, Klemens J. and Maniatis, Tom'
title: A systematic analysis of the factors that determine the strength of pre-mRNA splicing enhancers
---
authors: 'Graveley, Brenton R.'
doi: 10.1016/j.cell.2005.07.028
title: Mutually Exclusive Splicing of the Insect Dscam Pre-mRNA Directed by Competing Intronic RNA Secondary Structures
---
authors: 'Graveley, Brenton R.'
title: Sorting out the complexity of SR protein functions
---
authors: 'Gray, Nicola K. and Wickens, Marvin'
title: CONTROL OF TRANSLATION INITIATION IN ANIMALS
---
authors: 'Groban, Eli S. and Narayanan, Arjun and Jacobson, Matthew P.'
doi: 10.1371/journal.pcbi.0020032
title: Conformational Changes in Protein Loops and Helices Induced by Post-Translational Phosphorylation
---
authors: 'Gromer, Stephan and Johansson, Linda and Bauer, Holger and Arscott, L. David and Rauch, Susanne and Ballou, David P. and Williams, Jr., Charles H. and Heiner Schirmer, R. and Arner, Elias S. J.'
doi: 10.1073/pnas.2134510100
title: 'Active sites of thioredoxin reductases: Why selenoproteins?'
---
authors: 'MICHAEL GROMIHA, M.'
title: 'Structure Based Sequence Dependent Stiffness Scale for Trinucleotides: A Direct Method'
---
authors: 'Gruber, Andreas R and Bernhart, Stephan H and Hofacker, Ivo L and Washietl, Stefan'
doi: 10.1186/1471-2105-9-122
title: Strategies for measuring evolutionary conservation of RNA secondary structures
---
authors: 'GRUNDNER-CULEMANN, Elisabeth and MARTIN III, Glover W. and Harney, John W. and Berry, Marla J.'
title: Two distinct SECIS structures capable of directing selenocysteine incorporation in eukaryotes
---
authors: 'Grundner-Culemann, Elisabeth and Martin III, Glover W. and Tujebajeva, Rosa and Harney, John W. and Berry, Marla J.'
doi: 10.1006/jmbi.2001.4809
title: Interplay Between Termination and Translation Machinery in Eukaryotic Selenoprotein Synthesis
---
authors: 'Thakurta, Debraj Guha and Stormo, Gary D.'
title: Identifying target sites for cooperatively binding factors
---
authors: 'Guigo, Roderic and Dermitzakis, Emmanouil T. and Agarwal, Pankaj and Ponting, Chris P. and Parra, Genis and Reymond, Alexandre and Abril, Josep F. and Keibler, Evan and Lyle, Robert and Ucla, Catherine and Antonarakis, Stylianos E. and Brent, Michael R.'
doi: 10.1073/pnas.0337561100
title: 'Comparison of mouse and human genomes followed by experimental verification yields an estimated 1,019 additional genes'
---
authors: 'Guil, Sonia and Gattoni, Renata and Carrascal, Montserrat and Abian, Joaquín and Stevenin, James and Bach-Elias, Montse'
doi: 10.1128/MCB.23.8.2927-2941.2003
title: 'Roles of hnRNP A1, SR Proteins, and p68 Helicase in c-H-ras Alternative Splicing Regulation'
---
authors: 'Guillier, Maude and Gottesman, Susan'
doi: 10.1111/j.1365-2958.2005.04929.x
title: Remodelling of the Escherichia coli outer membrane by two small regulatory RNAs
---
authors: 'Gultyaev, Alexander P. and VAN BATENBURG, F. H. D. and Pleij, Cornelis W. A.'
title: An approximation of loop free energy values of RNA H-pseudoknots
---
authors: 'Guo, Peixuan'
title: 'RNA nanotechnology: Engineering, assembly and applications in detection, gene delivery and therapy'
---
authors: 'Gupta, Ramneek and Birch, Hanne and Rapacki, Kristoffer and Brunak, Søren and Hansen, Jan E.'
title: 'O-GLYCBASE version 4.0: a revised database of O-glycosylated proteins'
---
authors: 'Gupta, Arvind and Manuch, JÁN and Stacho, Ladislav'
title: Structure-Approximating Inverse Protein Folding Problem in the 2D HP Model
---
authors: 'Gupta, Shobhit and Zink, Dorothea and Korn, Bernhard and Vingron, Martin and Haas, Stefan A'
doi: 10.1186/1471-2164-5-72
title: Strengths and weaknesses of EST-based prediction of tissue-specific alternative splicing
---
authors: 'Gursinsky, Torsten and Jäger, Jana and Andreesen, Jan R. and Söhling, Brigitte'
doi: 10.1007/s002030000196
title: A selDABC cluster for selenocysteine incorporation in Eubacterium acidaminophilum
---
authors: 'Guyon, Isabelle and Weston, Jason and Barnhill, Stephen and Vapnik, Vladimir'
title: Gene Selection for Cancer Classification using Support Vector Machines
---
authors: 'Haas, Elizabeth S. and Armbruster, David W. and Vucson, Beverly M. and Daniels, Charles J. and Brown, James W.'
title: Comparative analysis of ribonuclease P RNA structure in Archaea
---
authors: 'Hackermqller, Jfrg and Meisner, Nicole-Claudia and Auer, Manfred and Jaritz, Markus and Stadler, Peter F.'
doi: 10.1016/j.gene.2004.11.043
title: 'The effect of RNA secondary structures on RNA-ligand binding and the modifier RNA mechanism: a quantitative model'
---
authors: 'Haggerty, Timothy J. and Zeller, Karen I. and Osthus, Rebecca C. and Wonsey, Diane R. and Dang, Chi V.'
doi: 10.1073/pnas.0931346100
title: A strategy for identifying transcription factor binding sites reveals two classes of genomic c-Myc target sites
---
authors: 'Halbeisen, R. E. and Galgano, A. and Scherrer, T. and Gerber, A. P.'
doi: 10.1007/s00018-007-7447-6
title: 'Post-transcriptional gene regulation: From genome-wide studies to principles'
---
authors: 'Hamada, Michiaki and Tsuda, Koji and Kudo, Taku and Kin, Taishin and Asai, Kiyoshi'
title: Mining frequent stem patterns from unaligned RNA sequences
---
authors: 'Hamilton, Russell S. and Hartswood, Eve and Vendra, Georgia and Jones, Cheryl and VAN DE BOR, Veronique and Finnegan, David and Davis, Ilan'
doi: 10.1261/rna.1264109
title: 'A bioinformatics search pipeline, RNA2DSearch, identifies RNA localization elements in Drosophila retrotransposons'
---
authors: 'Hammer, Brian K. and Bassler, Bonnie L.'
doi: 10.1073/pnas.0703860104
title: Regulatory small RNAs circumvent the conventional quorum sensing pathway in pandemic Vibrio cholerae
